Editor/Window/ConnectToFleetInput.cs

```diff
@@ -54,6 +59,11 @@ private async Task OnAnywhereConnectClicked(string text)
                 {
                     _fleetState = FleetStatus.Selected;
                 }
+                else
+                {
+                    _connectToAnywhereErrorBox.AddExternalButton(Urls.AwsIAMConsole, "View IAM console");
```

Comment:
Typically an API will throw an errorCode (or exception) with an appropriate message. IMO we should just pick the message from the exception and return it as is without having to deal with mapping. There are a number of service side issues that could happen, so it seems relevant that the Plugin just display with the service thinks is wrong.

However, if there is failure you expect and will only happen under a very certain cirumstance, please let me know what the circumstance is and we can get an appropriate error message.

Reply:
Also for the URL itself, what is the intention here? I would assume that any exception would point towards GameLift documentation with relevant instructions on how to resolve the error. Are there use cases outside of GameLift? We can discuss this during tomorrow's meeting!

Reply:
Got it! Well isn't the cw logs link something we already have rather than one that the product team will provide?

Reply:
I had added the logs link myself from what I could guess would be the correct link. This was mainly so I could make sure I wrote the correct code for `string.format` If we are happy with where it goes, it will not need changing. 
Currently goes to `"https://{0}.console.aws.amazon.com/cloudtrail/home?/events/events?"` with {0} being your region.

Reply:
Can we use the CW link instead for now? 

`https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logsV2:logs-insights`

Reply:
Thanks, changed to 
`https://{0}.console.aws.amazon.com/cloudwatch/home?region={0}#logsV2:logs-insights` should work with existing string.format